openstef/model/serializer.py

```diff
@@ -142,20 +143,31 @@ def _log_figures_with_mlflow(self, report) -> None:
 
     def load_model(
         self,
-        experiment_name: str,
+        pj: PredictionJobDataClass,
```

Comment:
This is not very backward compatible. Would it be better to add the run_id as optional?``
```suggestion
        experiment_name: str,
        model_run_id: typing.Optional[str] = None,
```

Reply:
yes good point, now changed this.